e2e/support/helpers/e2e-qa-databases-helpers.js

```diff
@@ -121,6 +121,40 @@ function recursiveCheck(id, i = 0) {
     });
     if (database.initial_sync_status !== "complete") {
       recursiveCheck(id, ++i);
+    } else {
+      recursiveCheckFields(id);
```

Comment:
This is the reason why it flakes on CI. Turns out the snapshot is created before all fields are synced. Despite having both `<db>.initial_sync_status === "complete"` and `<table>.initial_sync_status === "complete"`, some fields aren't fully synced. We can check by observing `last_analyzed`, if it's `null` it's not synced yet. But PK seems to always be null, so we skip checking that.